src/plugins/intel_gpu/include/intel_gpu/primitives/edgpsi.hpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
```

Comment:
Not important, but, as you can see from the folder, other experiemntal_... primitives' source codes files and their primitives are named with their full name so that is good to catch what they are from their names... edgpse is too compressed. Could you rename them so that the are better to be recognized?

Reply:
done (will be in new PR)